### PR TITLE
William/rn64

### PR DESCRIPTION
**Pull request by swansontec**
iOS feels weirdly faster with this change, but it could be my imagination.

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS Tablet
- [x] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -163,11 +165,12 @@ android {
         variant.outputs.each { output ->
             // For each separate APK per architecture, set a unique version code as described here:
             // https://developer.android.com/studio/build/configure-apk-splits.html
+            // Example: versionCode 1 will generate 1001 for armeabi-v7a, 1002 for x86, etc.
             def versionCodes = ["armeabi-v7a": 1, "x86": 2, "arm64-v8a": 3, "x86_64": 4]
             def abi = output.getFilter(OutputFile.ABI)
             if (abi != null) {  // null for the universal-debug, universal-release variants
                 output.versionCodeOverride =
-                        versionCodes.get(abi) * 1048576 + defaultConfig.versionCode
+                        defaultConfig.versionCode * 1000 + versionCodes.get(abi)
```
Comment (paullinator) [inline android/app/build.gradle:173]: Very bizarre change. Any reference on why?

**Review by paullinator — COMMENTED** [review]
Request insight on changes to build.gradle

File: android/app/build.gradle (hunk 1 above)
Comment (thehobbit85) [reply android/app/build.gradle:173]: https://github.com/facebook/react-native/pull/29808#issue-475784576

Comment (thehobbit85) [reply android/app/build.gradle:173]: https://github.com/software-mansion/react-native-reanimated/blob/d6ab6f3a28b9adcc410295b8f39fc73ce58afde2/Example/android/app/build.gradle#L137

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**